.github/workflows/release-please.yml

```
@@ -60,7 +63,7 @@ jobs:
     permissions:
       id-token: write
       contents: write
-    if: ${{ needs.release-please.outputs.release_created == 'true' }}
+    if: always() && needs.release-please.outputs.release_created == 'true'
```

```suggestion
    if: needs.release-please.outputs.release_created == 'true'
```

If always does what I think it does 😉

---

Unfortunately, this is a workaround on the dependency tree. By default, a job doesn't run if a dependency job doesn't run. With the `always()` you force it to run, even if the dependencies don't run, then the rest of the if statement decides whether to run it.